lib/ui/window.dart

```diff
@@ -116,9 +116,10 @@ class WindowPadding {
   }
 }
 
-/// An identifier used to select a user's language and formatting preferences,
-/// consisting of a language and a country. This is a subset of locale
-/// identifiers as defined by BCP 47.
+/// An identifier used to select a user's language and formatting preferences.
+/// This represents a [Unicode Language
+/// Identifier](https://www.unicode.org/reports/tr35/#Unicode_language_identifier),
+/// except variants have not been implemented yet..
```

Comment:
nit: `..`

Reply:
`except variants and extensions are not supported.` is probably best (no need to say "yet")